proto/proto/controller.proto

```diff
@@ -96,7 +97,7 @@ message ListEventbusResponse {
 }
 
 message GetEventbusWithHumanFriendlyRequest {
-  string namespace = 1;
+  uint64 namespace_id = 1;
```

Comment:
human-friendly why use namespace id? 
```suggestion
  string namespace = 1;
```

Reply:
metadata of eventbus doesn't have namespace field